### DISCUSSION TITLE
[feature] need center toast

### DISCUSSION THREAD

**Discussion by radiorz** [GeekyAnts/NativeBase#5482] [Feature request]
### Description

the toast has 6 positions but without center position
if possible please add the center postition to toast.

### Problem Statement

the toast has 6 positions but without center position
if possible please add the center postition to toast.

### Proposed Solution or API

 the toast has 6 positions but without center position
if possible please add the center postition to toast.

### Alternatives

https://www.npmjs.com/package/react-native-flash-message

**Comment by joe-yeomans** [comment]
@radiorz Could you not used an AlertDialog for this?
https://docs.nativebase.io/alert-dialog
